### ISSUE TITLE
find the min binlog offset  perhaps  has a bug  in increment stage  

### ISSUE THREAD

**Issue by xiangtao** [ververica/flink-cdc-connectors#329]
when all snapshot spit stage finished , it need to create a  MySqlBinlogSplit to start up  a increment binlog stage .  first need to find a startingOffset position from all assignedSnapshotSplit  of snapshot stage。but in MySqlHybridSplitAssigner.createBinlogSplit  , it seems wrong as ablove 
![image](https://user-images.githubusercontent.com/2025439/129852247-e6b3e1a2-e904-4ff9-8274-6e2a2c32f0ef.png)

it always  get BinlogOffset.INITIAL_OFFSET  value .  Finally, it is always read binlog from the beginning of mysql . 

to fix this bug  as follows 
```
BinlogOffset minBinlogOffset = null; 
    for (MySqlSnapshotSplit split : assignedSnapshotSplit) {
      // find the min binlog offset
      BinlogOffset binlogOffset = splitFinishedOffsets.get(split.splitId());
      if (minBinlogOffset == null || binlogOffset.compareTo(minBinlogOffset) < 0) {
        minBinlogOffset = binlogOffset;
      }
      finishedSnapshotSplitInfos.add(
          new FinishedSnapshotSplitInfo(
              split.getTableId(),
              split.splitId(),
              split.getSplitStart(),
              split.getSplitEnd(),
              binlogOffset));
      tableSchemas.putAll(split.getTableSchemas());
    }

```

**Comment by xiangtao** [comment]
@leonardBang

**Comment by leonardBang** [comment]
@xiangtao I think you're right, could you open a PR to fix this?

**Comment by xiangtao** [comment]
OK ，i will submit a pr @leonardBang

**Comment by xiangtao** [comment]
PR  :  https://github.com/ververica/flink-cdc-connectors/pull/331

**Comment by leonardBang** [comment]
fixed in master: e34ff84a43033b0909d62aaa1722ce322a354b0e
